src/openrct2/audio/audio.h

```diff
@@ -55,20 +55,21 @@ struct rct_ride_music_params
     uint16_t frequency;
 };
 
+struct Sound
+{
+    SoundId Id;
+    int16_t Volume;
+    int16_t Pan;
+    uint16_t Frequency;
+    void* Channel;
+};
+
 struct rct_vehicle_sound
 {
     uint16_t id;
```

Comment:
this is `VehicleId` can you do a refactor after this renaming it.

Reply:
Yes, I can :)